contributingGuides/HYBRID_APP.md

```diff
@@ -0,0 +1,212 @@
+# What is HybridApp?
+Every React Native mobile application needs native codebase with an entry point, known as `ReactRootView`. Depending on the platform the implementation differs. Nevertheless as a rule of thumb we can say that React Native just needs a **native** screen where JavaScript runtime can manipulate components and views.
+
+It means that whenever we create a React Native application from scratch we instantiate it on a (very basic, bare) native codebase. However, we can always use a different, already existing codebase to run React Native. This is how we've built HybridApp - we've created a new `ReactRootView`, and taken all JavaScript code to run it in so-called OldDot.
+
+It means that HybridApp is a **regular native application**, which has an additional screen that runs React Native. 
+# How is HybridApp built?
+If you have access to the `Mobile-Expensify` repository, you are eligible to build HybridApp. The main difference between NewDot and HybridApp is that the native code is located in a different place. The native code is located under `./Mobile-Expensify/Android` and `./Mobile-Expensify/iOS
+
+It is vital to understand, that modifying any code in `./android` and `./ios` folders in the root of the project **will not affect the HybridApp build at all.** In this case, if you'd like to change native code, you have to go to `./Mobile-Expensify/Android` and `./Mobile-Expensify/iOS` first. This is **especially important** when you want to remove native cache or `Pods` manually. 
+# How do the scripts work?
+While working on the HybridApp we've prepared a set of scripts to make the work with HybridApp easier. In general, the scripts `npm install`, `npm run clean`, `npm run android`, and `npm run ios` work on the following condition:
+
+**If you have cloned HybridApp to `Mobile-Expensify` submodule, change the default behaviour of the scripts to build HybridApp instead of NewDot.**
```

Comment:
If you have cloned HybridApp to `Mobile-Expensify` submodule, the default behaviour of the scripts changes to build HybridApp instead of NewDot